src/styles.css

```diff
@@ -31,10 +31,13 @@ h1, h1, h3, h4, h5, h6 {
   text-align: center;
 }
 
-.bg-default {
+/* .bg-default {
   @apply bg-secondary text-primary;
-}
+} */
 
+.github-corner {
```

Comment:
These styles are not applied because of css specificity. The github log is an SVG with inline styles:

Examples of inline styles:
<img width="669" alt="Screen Shot 2020-09-26 at 8 14 00 AM" src="https://user-images.githubusercontent.com/6374832/94343896-3ecb1b00-ffd0-11ea-8b94-7e36bfb5384f.png">

Inline styles have a higher precedence in deciding the styles of an element in case there are conflicting styles. I recommend reading about this more because its a very important topic in CSS:

https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity
https://www.smashingmagazine.com/2007/07/css-specificity-things-you-should-know/
https://dev.to/emmabostian/css-specificity-1kca
https://css-tricks.com/specifics-on-css-specificity/
https://specificity.keegan.st/
https://cssspecificity.com/